src/OrchardCore/OrchardCore.Data.Abstractions/OrchardCore.Data.Abstractions.csproj

```diff
@@ -14,6 +14,10 @@
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="YesSql.Abstractions" Version="3.0.12" />
```

Comment:
We already separate beteen `Data.Abstractions` & `Data.YesSql.Abstractions`, here you bring a dependency on YesSql again, which is a not good idea